var/spack/repos/builtin/packages/icedtea/package.py

```diff
@@ -58,7 +58,7 @@ class Icedtea(AutotoolsPackage):
     depends_on('libxcomposite', when='~X', type='build')
     depends_on('libxau', when='~X', type='build')
     depends_on('libxdmcp', when='~X', type='build')
-    depends_on('gtkplus+X', when='~X', type='build')
```

Comment:
I assume that, incidentally, we fixed a bug :+1:

Reply:
I think that the `+X` deps are needed anyway, they all become `build` only when `~X` is requested on `icedtea`:

```
# X11 deps required for building even when headless
```